django/db/backends/oracle/operations.py

```diff
@@ -202,7 +202,7 @@ def get_db_converters(self, expression):
         # Oracle stores empty strings as null. If the field accepts the empty
         # string, undo this to adhere to the Django convention of using
         # the empty string instead of null.
-        if expression.field.empty_strings_allowed:
+        if hasattr(expression, 'field') and expression.field.empty_strings_allowed:
```

Comment:
Wonder if we should check for `output_field` instead?

```suggestion
        if expression.output_field.empty_strings_allowed:
```